fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/CallOperator.java

```diff
@@ -49,6 +49,9 @@ public class CallOperator extends ScalarOperator {
     // The flag for distinct function
     private boolean isDistinct;
 
+    // The flag for remove distinct agg func because add extra agg steps in SplitMultiPhaseAggRule
+    private boolean removedDistinct;
```

Comment:
when we don't remove the distinct flag for a distinct function?

Reply:
distnct flag has been reomoved in the rewrite aggCall process. We have to use the `removedDistinct` to distinguish the call without distinct flag orginal from the call with distinct flag but had been removed the flag because of split agg.